app/javascript/src/assets/scss/widgets/_conversation-view.scss

```diff
@@ -119,14 +122,15 @@
     &.left {
       .bubble {
         background: $white;
-        color: $color-heading;
-        margin-right: auto;
         border-bottom-left-radius: 0;
         border-top-left-radius: 0;
+        color: $color-heading;
+        margin-right: auto;
       }
 
-      &+.right {
+      +.right {
```

Comment:
@nisusam `&` is omitted here. Is there any reason for it ? Thanks

Reply:
@nithindavid The following error I was getting after running `scss-lint`

```
UnnecessaryParentReference: Unnecessary parent selector (&)
```

Reply:
Looks like `&` is not needed. Thanks :)